java/src/main/java/ai/rapids/cudf/HashType.java

```diff
@@ -23,9 +23,7 @@
  */
 public enum HashType {
   IDENTITY(0),
-  MURMUR3(1),
-  HASH_SPARK_MURMUR3(2),
-  HASH_MD5(3);
+  MURMUR3(1);
```

Comment:
Quick question.  I see the MD5 type being removed here but still being referenced elsewhere (ColumnVector.java). Is that intended?

Reply:
References to `HashType.HASH_MD5` should all be removed from `ColumnVector.java` in this PR.
https://github.com/rapidsai/cudf/pull/15375#discussion_r1541296930